diff -ruN google-ads-9.0.0/setup.py google-ads-9.0.0-patch/setup.py
--- google-ads-9.0.0/setup.py	2021-02-17 13:06:02.000000000 +0100
+++ google-ads-9.0.0-patch/setup.py	2021-02-26 08:30:12.894316086 +0100
@@ -23,8 +23,7 @@
     "grpcio >= 1.33.2, < 2.0.0",
     "protobuf >= 3.13.0, < 4.0.0",
     "PyYAML >= 5.1, < 6.0",
-    "setuptools>=40.3.0",
-    "pep562 >= 1.0, < 2.0",
+    "setuptools>=40.3.0"
 ]
 
 tests_require = [
